NetworkingSampleApp/NetworkingSampleApp/Scenes/Download/DownloadsViewModel.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  DownloadsViewModel.swift
+//  
+//
+//  Created by Matej Molnár on 07.03.2023.
+//
+
+import Foundation
+import Networking
+import OSLog
+
+extension DownloadAPIManager {
```

Comment:
I don't see any additional value having it in separated extension in this class, for sample we can init the DownloadManager in DownloadsViewModel

Reply:
The download manager instance is used in multiple places. I agree that it does not make sense having this extension in the DownloadsViewModel so I moved it to a separate file.